src/main/java/training/busboard/Models/StopPoint.java

```diff
@@ -0,0 +1,23 @@
+package training.busboard.Models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+public class StopPoint {
+
+    public String naptanId;
+    public float distance;

```

Comment:
As for `timeToStation`, some units would be helpful here, even if it means you have to add another Jackson annotation to match this field to the JSON property.